api/include/opentelemetry/trace/propagation/detail/hex.h

```diff
@@ -43,13 +43,13 @@ bool IsValidHex(nostd::string_view s)
  */
 bool HexToBinary(nostd::string_view hex, uint8_t *buffer, size_t buffer_size)
 {
+  std::memset(buffer, 0, buffer_size);
```

Comment:
Also check the input `hex` is empty, as it cannot be handled in below code?

Reply:
It is handled - nothing is written to the buffer if hex is empty.